3rd-party-software/DSE_5.1.31_3rd_party_software.md

```diff
@@ -0,0 +1,561 @@
+# DataStax Enterprise 5.1.31 third-party software
+
+| Component  |  Versions  | Licenses  |
+|:-----------|:----------:|:----------|
+| airline | 0.6 | UNKNOWN |
```

Comment:
There is a ton of `unknown` licences, I don't know why it is like that (at least right now) but it does not look good